spyder_unittest/widgets/configdialog.py

```diff
@@ -93,6 +94,22 @@ def __init__(self, frameworks, config, parent=None):
 
         layout.addSpacing(20)
 
+        pyexec_label = QLabel(_('Use the following Python interpreter:'))
```

Comment:
I strongly disagree with adding an option for this because it'd made the user interface and experience between this plugin and others in Spyder inconsistent. That's because users would be able to select a Python interpreter here and another one in Preferences, and the last one is shown in the status bar as well.

So it'd be really confusing for users to know which interpreter is currently in use for this plugin, especially if they forgot that they set a different one through this option.

Reply:
Yea that's fine with me. I didn't know how to get the value from the preferences, so I just added a manual way to change the executable. I can remove all of that, but I'll wait to see your reply above.